psi-ms.obo

```diff
@@ -22655,6 +22655,386 @@ relationship: has_metric_category MS:4000012 ! single run based metric
 relationship: has_value_type xsd:int ! The allowed value-type for this CV term
 relationship: has_units UO:0000189 ! count unit
 
+[Term]
+id: MS:4000108
+name: Peak density distribution MS1 Q1, Q2, Q3
+def: "From the distribution of peak densities in MS1, the quartiles Q1, Q2, Q3" [PSI:MS]
+comment: The distribution of peak densities in MS1 can provide insight into the presence of nuisance factors.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000167 ! first quartile
+relationship: has_value_concept STATO:0000574 ! center value
+relationship: has_value_concept STATO:0000170 ! third quartile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000109
+name: Peak density distribution MS1 mean
+def: "From the distribution of peak densities in MS1, the mean" [PSI:MS]
+comment: The distribution of peak densities in MS1 can provide insight into the presence of nuisance factors.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000110
+name: Peak density distribution MS1 sigma
```

Comment:
**suggestion:** Rename "sigma" to "standard deviation" to be more clear?

Reply:
I'd favour a short name and let `relationship: has_value_concept STATO:0000237 ! standard deviation` do the specification, but my heart's not in it.

Reply:
i.e. you decide